src/common/components/Textarea/Textarea.theme.ts

```diff
@@ -0,0 +1,39 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TextareaVariants = VariantProps<typeof textareaTheme>;
+
+export const textareaTheme = tv(
+  {
+    slots: {
+      wrapper: [],
+      textarea: [
+        "flex",
+        "items-start",
+        "p-2",
+        "w-80",
+        "max-w-[30rem]",
+        "min-h-32",
+        "gap-2",
+        "rounded-lg",
+        "border",
+        "border-solid",

```

Comment:
Just a very minor nitpick. I think it defaults to solid!

Reply:
removed at [6124c5c](https://github.com/AfterClass-io/afterclass.io-v2/pull/82/commits/6124c5ca816313306a720d3c70c1d7318b9cf49b)!